src/style.css

```diff
@@ -1,3 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+
+  --activation-color: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  margin: 10px 50px;
```

Comment:
remove it

Reply:
if don't set the height, extra space will appear below the image